### ISSUE TITLE
Game Maker Studio support?

### ISSUE THREAD

**Issue by psychoanima** [DragonBones/DragonBonesAS#76]
Hi,

I just noticed that 2.3 version of DragonBones can import Spine files. 

Game maker studio is supporting Spine json format. Is it possible to make DragonBones to export spine like json format so it can be imported into Game Maker Studio? That would be awesome feature and money saver for us indie game developers!

**Comment by blurymind** [comment]
does game maker studio 2 support spine?

**Comment by bigtunacan** [comment]
@blurymind Yes.  GM:Studio 2 has native support for importing Spine's json format.  If you go to http://docs2.yoyogames.com/ and then search for Spine you can then see additional details under 

Importing Non-Bitmap Sprites
    - Skeletal Animation Sprites (Spine)

**Comment by blurymind** [comment]
Can it also import dragonbones json data?
I dont own a spine license

**Comment by bigtunacan** [comment]
@blurymind No.  It supports Spine json, but you need a Spine license due to licensing resctrictions.

It also supports swf and sprite strips.

**Comment by blurymind** [comment]
that is precisely why i think it would be very beneficial if gamemaker
studio 2 adds support for dragonbones json data. It will make it more
afordable for indie devs to use this kind of animation

**Comment by blurymind** [comment]
even better it would be if godot engine had support for dragon bones..
but alas, there is only a community made spine module
